react/components/SettingsContext.ts

```diff
@@ -0,0 +1,5 @@
+import { Context, createContext } from 'react'
+
+const SettingsContext: Context<any> = createContext({})
```

Comment:
This is a `Record<string, unknown>`, right? Or something like that

Reply:
Right! I will change that :)